source/tyr/tyr/command/import_last_dataset.py

```diff
@@ -49,6 +49,9 @@ def import_last_dataset(
 
     """
     instance = models.Instance.query_existing().filter_by(name=instance_name).first()
+    # little trick to keep a flag on the command line: -a enable mimir import,
+    # we reverse it to match import_data
+    skip_mimir = not allow_mimir
```

Comment:
Minor: `need_mimir` would be more understandable than `allow_mimir` for me

Reply:
or maybe just use what it's in the comment: `enable_mimir_import`?
So you can have its equivalent: `disable_mimir_import = not enable_mimir_import` :)

Reply:
I didn't want to use *enable* as it doesn't enable anything, if mimir isn't setup passing this parameter won't do anything

Reply:
Ok, I see where you're going. Can't find anything better unfortunately :(